templates/simple/script.js

```diff
@@ -0,0 +1,15 @@
+function toggleScreenshot(e){
+    if(this.innerText === "Screenshot -"){
+        this.innerText = "Screenshot +";
+        this.nextElementSibling.style.display = "none";
+    } else {
+        this.innerText = "Screenshot -";
+        this.nextElementSibling.style.display = "block";
+    }
+}
+
+var toggleEls = document.querySelectorAll("a.toggle");
+
+for(var i = 0; i < toggleEls.length; i++){
+    toggleEls[i].addEventListener("click", toggleScreenshot);
+}
```

Comment:
Can you also split the [condition](https://github.com/gkushang/cucumber-html-reporter/blob/develop/lib/reporter.js#L132) to use this particular `script.js` while generating the report?